crosstool/wrapped_clang.cc

```diff
@@ -359,6 +360,14 @@ void ProcessArgument(const std::string arg, const std::string developer_dir,
     return;
   }
 
+  // Don't strip the binary directly when stripping is requested, as this will
+  // remove all debug information that will be required to generate a .dSYM
+  // bundle
+  if (arg == "-Wl,-S") {
```

Comment:
i think we can improve this by using an env var instead, but we can do that separately